.github/workflows/binary.yaml

```diff
@@ -26,6 +26,9 @@ jobs:
             name: x86_64-unknown-linux-musl
             installable: .#dune-static-experimental
 
+    # If the latest commit is the same as latest run, don't re-run.
+    if: (git log --format=%H -1) != (jq -r ".|sort_by(.date)|last|.commit" < metadata.json)
```

Comment:
I was curious what this does, but generally I think it is better to use `git rev-parse HEAD`.

Reply:
It works indeed, but I'm not sure I'm a fan. The manpage for `git-rev-parse` is very unclear about what it does ("massage parameters", "git porcelainish commands", only saying what it does with a simple argument without options 200 lines in), whereas the log command follows the comment and the jq script pretty nicely: take the last commit, and print only its full hash. I suppose `git log -1 --format=%H` would be even more samey for that purpose.
All of this is just my opinion of course

Reply:
Counterpoint: we already use `git rev-parse HEAD` throughout the same file. I'll homogenize.

Reply:
The thing is from `git log` I expect the log of commits (and then I have to understand what `%H` is) whereas `rev-parse` is a fairly common idiom and we use that basically everywhere in the Dune codebase where we want to know the commit-hash of a rev.

I do agree that the manpage of `git-rev-parse` is absolute bonkers. "Pick out and massage parameters" like what's that even supposed to mean. I guess the reason is that `rev-parse` is part of the "plumbing" and these commands just exist to enable the porcelain on top. I looked whether `git describe` can be used to get a commit hash but no, apparently it will do everything but that.